src/EnergyPlus/HVACManager.cc

```diff
@@ -303,7 +302,9 @@ namespace HVACManager {
 
         // SUBROUTINE PARAMETER DEFINITIONS:
         static ObjexxFCL::gio::Fmt EndOfHeaderFormat("('End of Data Dictionary')");          // End of data dictionary marker
+        static constexpr auto EndOfHeaderString("End of Data Dictionary");          // End of data dictionary marker
         static ObjexxFCL::gio::Fmt EnvironmentStampFormat("(a,',',a,3(',',f7.2),',',f7.2)"); // Format descriptor for environ stamp
+        static constexpr auto EnvironmentStampFormatStr("{},{},{:7.2F},{:7.2F},{:7.2F},{:7.2F}\n"); // Format descriptor for environ stamp
```

Comment:
@lefticus is there a gain to using `auto` here?  I like using it for references so that we don't have to spell out a long type name, but in cases like this, I kinda like being intentional.  It should just be a `std::string`, right?  

This doesn't need to be fixed before this PR merges, but I'd like to understand if there is an alternative benefit from using `auto` before it is used more widespread.

Reply:
@Myoldmopar   Interesting one for you to bring up. The actual type is `const char *`, which is why I didn't spell it out.

using `std::string` for the type would actually cause a performance hit because it cannot be `constexpr` and would cause a static initialization check on each access to the value.

For the moment I'm leaving them as `auto` for place holders with the hope that if we upgrade the build system to C++14 or greater (I haven't done that yet) I can use `constxpr fmt::string_view` which means the option of some compile-time checking of parameters as well as better perf at runtime.

Reply:
That makes sense, thanks for the clarification.  As for moving to C++14, Linux and Mac should be no problem, right?  Is there a specific version of Visual Studio we need to target?

Reply:
@Myoldmopar 2019 would be amazing, 2017 would be awesome. But in general I think it's time for a refresh to set the minimum supported compilers so we can move forward in places where it makes sense to, even with gcc/clang.